App/BraveWidgets/FavoritesWidget.swift

```diff
@@ -99,7 +99,8 @@ private struct FavoritesGridView: View {
       return 8
     case .systemLarge:
       return 16
-    case .systemSmall, .systemExtraLarge:
+    case .systemSmall, .systemExtraLarge, .accessoryCircular,
+        .accessoryRectangular, .accessoryInline:
```

Comment:
Won't this fail to compile without runtime availability checks? The `accessory*` families are iOS 16+

Reply:
strange it does compile fine for me hmm. But good point I missed the iOS 16 requirement